.github/scripts/smoke_test.sh

```diff
@@ -0,0 +1,78 @@
+#! /bin/bash
+
+# Software test
+test_software() {
+cd ../../.. && \
+if [ -d "./$project" ]; then
+    rm -r $project
+fi
+dbt init -s $project && \
+{ cat > ./.dbt/profiles.yml << EOF
+$project:
+  outputs:
+    dev:
+      password: $test_password
+      port: 9047
+      software_host: $test_host
+      threads: 1
+      type: dremio
+      use_ssl: false
+      user: $test_user
+  target: dev
+EOF
+} && \
+
+cd $project/ && \
+dbt debug && \
+dbt run
+
+}
+
+#Cloud test
+test_cloud() {
+cd ../../.. && \
+if [ -d "./$project" ]; then
+    rm -r $project
+fi
+dbt init -s $project && \
+{ cat > ./.dbt/profiles.yml << EOF
+$project:
+  outputs:
+    dev:
+      cloud_host: $test_host
+      cloud_project_id: $test_cloud_project
+      pat: $test_token
+      threads: 1
+      type: dremio
+      use_ssl: true
+      user: $test_user
+  target: dev
+EOF
+} && \
+echo 'vars:' >> ./$project/dbt_project.yml && \
```

Comment:
Should we add this variable in software as well? Both are affected when testing with a non-admin account.

Reply:
Good point. I'll add it in.